test/event_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/event'
+
+class EventTest < Minitest::Test
```

Comment:
Overall, these tests look good - don't forget to test the helper methods you create!